src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -88,7 +88,7 @@ private static DetailNode parseJavadocAsDetailNode(String javadocComment) {
      * @param parseErrorMessage ParseErrorMessage
      * @return error message
      */
-    private static String getParseErrorMessage(ParseErrorMessage parseErrorMessage) {
+    public static String getParseErrorMessage(ParseErrorMessage parseErrorMessage) {
```

Comment:
I only see tests referrencing this method.
We should not loosen modifiers just for easier testing. Use reflection to grab method.